packages/sol-rayz/src/getParsedNftAccountsByCollection.ts

```diff
@@ -0,0 +1,33 @@
+// @ts-nocheck
+import { clusterApiUrl, Connection, Commitment, PublicKey } from '@solana/web3.js';
+import { METADATA_PROGRAM } from './config/metaplex';
+import { decodeTokenMetadata } from './utils';
+
+export const createConnectionConfig = (clusterApi = clusterApiUrl('mainnet-beta'), commitment = 'confirmed') =>
+  new Connection(clusterApi, commitment as Commitment);
+
+export const getParsedNftAccountsByUpdateAuthority = async ({
```

Comment:
file name different from the method name. is it intentional?
I propose name file same as method

Reply:
+